crown-court-proceeding/build.gradle

```diff
@@ -83,14 +83,23 @@ dependencies {
     testImplementation "org.pitest:pitest:$versions.pitest"
     testImplementation "com.github.tomakehurst:wiremock-jre8-standalone:$versions.wiremockVersion"
 
-}
+    testImplementation 'org.testcontainers:localstack:1.18.3'
+    testImplementation 'cloud.localstack:localstack-utils:0.2.23'
+//    testImplementation group: 'com.github.tomakehurst', name: 'wiremock', version: '2.27.2', ext: 'pom'
+
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk', version: '1.12.264'
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk-sqs', version: '1.12.264'
```

Comment:
Could you please make the below as testImplementation as they are used only in the integration tests:
    implementation group: 'com.amazonaws', name: 'aws-java-sdk', version: '1.12.264'
    implementation group: 'com.amazonaws', name: 'aws-java-sdk-sqs', version: '1.12.264'